backend/src/infra/socket/socket.manager.ts

```
@@ -2,23 +2,32 @@ import { Server as HttpServer } from 'http';
 import { Server as SocketIOServer, Socket } from 'socket.io';
 import logger from '@/utils/logger.js';
 import { TokenManager } from '@/infra/security/token.manager.js';
-import { ServerEvents, ClientEvents, SocketMetadata, NotificationPayload } from '@/types/socket.js';
+import {
+  ServerEvents,
+  ClientEvents,
+  SocketMetadata,
+  NotificationPayload,
+} from '@/types/socket.js';
```

ESLint

[error] Replace `⏎··ServerEvents,⏎··ClientEvents,⏎··SocketMetadata,⏎··NotificationPayload,⏎` with `·ServerEvents,·ClientEvents,·SocketMetadata,·NotificationPayload·`

(prettier/prettier)